.github/workflows/integration_test.yaml

```diff
@@ -300,6 +321,26 @@ jobs:
           EXPERIMENT_NAME: ${{ inputs.chaos-experiments }}
           TOTAL_CHAOS_DURATION: ${{ inputs.chaos-duration }}
           KUBE_CONFIG_DATA: ${{ env.KUBE_CONFIG_DATA }}
+      - name: Install k6s
+        if: ${{ inputs.load-test-enabled }}
+        run: sudo snap install k6
+      - name: Run k6 load tests
+        if: ${{ inputs.load-test-enabled }}
+        run: k6 run load_tests/load-test.js ${{ inputs.load-test-run-args }}
+      - name: Run Github Trivy FS Action
+        if: ${{ inputs.trivy-fs-enabled }}
+        uses: aquasecurity/trivy-action@master
+        with:
+          scan-type: 'fs'
+          scan-ref: ${{ inputs.trivy-fs-ref }}
+          trivy-config: ${{ inputs.trivy-fs-config }}
+      - name: Run Github Trivy Image Action
+        if: ${{ inputs.trivy-image-enabled && inputs.trivy-image-ref != '' }}
```

Comment:
Is this the desired behaviour? If so, can we get rid of the param `trivy-image-enabled`?

Reply:
You're right, there is no need of trivy-image-enabled, removing.